### PR TITLE
Host reference manual on ocaml.org instead of caml.inria.fr

### PR DESCRIPTION
**Pull request by xavierleroy**
Following COVID-19 containment measures, Inria Paris is closed, and there is a possibility that the facility will be shut down completely in the near future, including the server room that contains the caml.inria.fr Web server.

It is therefore high time to stop hosting crucial resources for the OCaml community on caml.inria.fr, as I have repeatedly argued in the past.  

This PR adds a copy of the reference manual (various formats, various versions) to ocaml.org and rewrites URLs all over so that the reference manual is accessed from ocaml.org and the source tarballs from github.com.  

The PR is best reviewed commit by commit.  The first commit is huge (it's the import of all the files for the reference manual) but the other commits have reasonable size.

I do not know how to rebuild and test the Web site locally, so all this is untested.  Please tell me how to test.

Special mention: @Octachron @pmetzger

### CODE REVIEW

**Comment by xavierleroy** [comment]
I was able to build and install locally, and run a link checker.  There's a bunch of broken links, but not really related to this PR, e.g. they were there before in the ocaml.org Website or in the "single HTML page" version of the manual.

**Comment by avsm** [comment]
Thanks; I'm also reprovisioning ocaml.org's hosting provider at the moment, so I'll test this locally in the next few hours.

**Comment by Chris00** [comment]
Thanks @avsm — I have absolutely no time until tomorrow afternoon, moving my courses online and helping colleagues to do the same has been a very time consuming!

**Comment by avsm** [comment]
I can imagine @chris00, good luck with that! We're just shutting down Cambridge University as well.

**Review by avsm — APPROVED** [review]
Read through it, will merge after a test deployment has completed.

**Comment by Octachron** [comment]
On the compiler side, once this is merged, I will update the instructions for deploying the manual in the compiler release checklist.

File: Makefile [hunk 1]
```diff
@@ -25,7 +26,7 @@ syncotherfiles:
 	 rsync --exclude '*~' --exclude '*.md' \
 	  --exclude '*.cmi' --exclude '*.cmo' --exclude '*.annot' \
 	  -rltHprogv ../ocaml.org-media/* ocaml.org/ ; fi
-	rsync -av --delete site/releases/?.?? ocaml.org/releases/
+	rsync -av --delete $(MANUAL_DIRS) ocaml.org/releases/
```
Comment (avsm) [inline Makefile]: The `-v` is pretty verbose in the deployment logs, but I'll quieten it down at a later stage.

Comment (xavierleroy) [reply Makefile]: No problem, I'll remove the -v.  More generally: can I squash and cleanup the history before you merge?

Comment (avsm) [reply Makefile]: Sure -- I'm knee deep in VM provisioning, so no rush here.

**Comment by xavierleroy** [comment]
I cleaned up the history.  As fas as I'm concerned, this is ready for merging.  Thanks @avsm for the quick feedback!

**Comment by avsm** [comment]
Thanks @xavierleroy! Should be live in an hour or so.

**Comment by Chris00** [comment]
On 18 March 2020 at 15:27 CET, Anil Madhavapeddy <notifications@github.com> wrote:
>
> I can imagine @Chris00, good luck with that! We're just shutting down
> Cambridge University as well.

Good luck to you (and all others) too!

**Pull request merged**